### DISCUSSION TITLE
怎么使用注册邮箱服务？

### DISCUSSION THREAD

**Discussion by zadishu** [qd-today/qd#454] [Q&A]
![WeChat截图_20230724145807](https://github.com/qd-today/qd/assets/110985414/114422e3-35d6-4e3e-8292-c6f546a72c08)
怎么设置都无法使用，帮我看看哪里出错

**Comment by a76yyyy** [comment]
https://qd-today.github.io/qd/zh_CN/toolbox/pusher.html#%E9%82%AE%E4%BB%B6%E6%8E%A8%E9%80%81

**Reply by zadishu** [reply]
我对这个研究了很久，一直收不到注册邮件

**Reply by a76yyyy** [reply]
你看看是不是邮箱那边没有设置开启smtp

**Reply by zadishu** [reply]
开启了，你看看我的参数对不对
domain的网址带端口的问题？

**Reply by zadishu** [reply]
(535, b'Error: authentication failed')
提示这个，密码没问题

**Reply by a76yyyy** [reply]
可能是smtp不能用原始密码，需要用授权码之类的

**Reply by zadishu** [reply]
是用的授权码，一直发送不成功

**Reply by a76yyyy** [reply]
参考这个https://blog.csdn.net/leona_leejj/article/details/91988010

https://jingyan.baidu.com/article/c275f6ba33a95de33d7567d9.html

你看看是不是哪一步漏了

或者重置一下授权码
